### PR TITLE
Corrections/chapter cues

### PR DESCRIPTION
**Pull request by icke-siegen**
Correction in chapter 'Cues'. Please see commit annotations for details.

### CODE REVIEW

**Comment by congoblue** [comment]
These are good corrections. 
It would be good to change **Swop** references to say **Select** as we are going to refer to the buttons as "Select" from now on.
Also in cue-playback.md it would be good to have the fader hold settings \[Always Hold\] etc rendered as softkey options
Are you happy for me to edit your changes or do you want to do it?

**Comment by icke-siegen** [comment]
> These are good corrections.
> It would be good to change **Swop** references to say **Select** as we are going to refer to the buttons as "Select" from now on.
> Also in cue-playback.md it would be good to have the fader hold settings [Always Hold] etc rendered as softkey options
> Are you happy for me to edit your changes or do you want to do it?

Of course - go ahead and edit them :)

**Review by congoblue — DISMISSED** [review]

File: docs/cues/cue-playback.md [hunk 1]
```diff
@@ -86,33 +86,45 @@ graphic you can jump to a page number by entering the number.
...
-    [\[Playback Paging\] user setting](../system-settings/user-settings.md#playback-paging).
+    blue. 
+
+The setting \[Normal\] gives the default state, for consoles 
+without motorised faders this is \[Always Hold\].
 
 ### Consoles with Motorised Faders
 
--   Playbacks which are fired when you change page remain active, but
+-   By default, playbacks which are fired when you change page remain active, but
```
Comment (Farrser) [inline docs/cues/cue-playback.md:116]: I think that the playback will always remain active? What changes is whether it pages?

Comment (congoblue) [reply docs/cues/cue-playback.md:116]: Yes that is now a bit confusing. I will rewrite this section again to make that clearer

**Review by congoblue — APPROVED** [review]
Update section about playback paging

**Review by Farrser — APPROVED** [review]

**Pull request merged**